local-setup.sh

```diff
@@ -1,117 +1,63 @@
 #!/bin/bash
 
-# setup local ubuntu enviornment
-
-if [ $(sudo apt update && apt list --upgradable | wc -l) -le 1 ]
+# curl should be installed
+if (curl --version)
 then
-  echo "System already up to date"
+  echo "**curl already installed"
```

Comment:
What are the asterisks for? If you're trying to differentiate the messages maybe look up how to change the color of an echo statement and use Green for "already" and Yellow for "doing"?

Reply:
Sounds good. I'll add that.